_posts/2024-08-31-summer-2024-reu-presentations.md

```diff
@@ -0,0 +1,24 @@
+---
+date: 2024-08-31 00:00:00-07:00
+description: null
+featuredImage: assets/reu/2024-08-29-Final-Presentations.png
+author: Nathan
+layout: blog-post
+slug: /summer-2024-reu-presentations
+title: Summer 2024 Final Presentations
+categories:
+- news-and-updates
+tags:
+- reu
+- reu2024
+- summer
+---
+<iframe width="896" height="504" src="https://www.youtube.com/embed/dwnrstHYuwM?si=_NGEEn4SNX2CnhIj" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
```

Comment:
I need to check what this looks like, but I reccomend seeing with="100%" I noticed this was an issue in SDZWA Presentation iframe's